lib/src/components/NxTableCell/NxTableCell.tsx

```diff
@@ -58,6 +71,30 @@ const NxTableCell = function NxTableCell(props: Props) {
     maskedSort = <NxFontAwesomeIcon icon={faSort} />;
   }
 
+  const filterContent = onFilterChange && (
+    <NxFilterInput disabled={isFilterDisable}
+                   placeholder={filterPlaceholder || ''}
+                   onChange={onFilterChange}
+                   value={filter || ''}/>
+  );
+
+  const onDropdownLinkClick = (item: string) => {
+    if (onDropdownLinkChange) {
+      onDropdownLinkChange(item);
+    }
+  };
+
+  const renderedDropdownOptions = filterDropdownOptions && Array.from(filterDropdownOptions).map((item: string) => {
+    return (
+      <NxButton key={item} className="nx-dropdown-button" onClick={() => onDropdownLinkClick(item)}>{item}</NxButton>
+    );
+  });
+
+  const filterDropdownElement =
+    <NxStatefulDropdown className="nx-dropdown--navigation" label={filterDropdownLabel || ''}>
```

Comment:
There are a couple of things that could be used to implement this sort of filter:

* `NxDropdown`, as you have here.  This isn't really the right one to use as it it intended to represent a dropdown menu of links or action buttons and not a selection box.
* `<select>`.  This is what @glenhunter and I recommended to you the other day.  There aren't RSC styles for this yet, so you'd have to create them.
* `NxFilterInput` combined with a `<datalist>`.  @glenhunter (re)discovered `<datalist>` within the last few days, and it seems like it could be an excellent fit for this usecase.  Since the UI would be an `NxFilterInput`, little or no new styling would be needed, though the ability to pass a datalist id would need to be added to `NxFilterInput`.

Reply:
> NxFilterInput combined with a <datalist>. @glenhunter (re)discovered <datalist> within the last few days, and it seems like it could be an excellent fit for this usecase. Since the UI would be an NxFilterInput, little or no new styling would be needed, though the ability to pass a datalist id would need to be added to NxFilterInput.

We just discussed in WW and we would like to go with this option.  Basically, the filter will always be an `NxFilterInput` and a given instance may or may not (at the designers discretion) have a `<datalist>` associated with it giving it auto-suggestion capability.

Reply:
No styles yet for the filter with datalist: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/4f2e827522b2c9ccfec400e47e4f3529cb43259d

![image](https://user-images.githubusercontent.com/57679720/95900361-1134ee80-0d92-11eb-92c0-4a6d899edd4b.png)

![image](https://user-images.githubusercontent.com/57679720/95900373-15f9a280-0d92-11eb-9a09-d9fc736f1a4b.png)

![image](https://user-images.githubusercontent.com/57679720/95900391-201ba100-0d92-11eb-8b5b-d08c06ce4951.png)

Reply:
I'll take the changes from @glenhunter PR to handle this: https://github.com/sonatype/sonatype-react-shared-components/pull/166

Reply:
Changes from NxFilterInput with the list here: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/54bb90c8464c837b780bb0dc1ed726103de00243